code/game/turfs/open/openspace.dm

```diff
@@ -23,7 +23,6 @@ GLOBAL_DATUM_INIT(openspace_backdrop_one_for_all, /atom/movable/openspace_backdr
 	smoothing_groups = SMOOTH_GROUP_FLOOR_OPEN_SPACE
 	smoothing_list = SMOOTH_GROUP_OPEN_FLOOR + SMOOTH_GROUP_CLOSED_WALL
 	neighborlay_self = "staticedge"
-	turf_flags = TURF_WEATHER_PROOF
```

Comment:
check with snow to see if snow builds up on openspace, this was an issue which led to this being added at like 3am